### DISCUSSION TITLE
Is there a way to get poseHeading data from the viewer?

### DISCUSSION THREAD

**Discussion by AndyBaird** [mistic100/Photo-Sphere-Viewer#1175] [General]
**Use case:**
I want the user to be able to pan around an image and to "save" the angle they are looking at so the next time then load that image it initializes with the same perspective.

**Attempts:**
I feel like I am missing something obvious and hoping someone can point it out. Using the docs I can see how to get position data, but it is in yaw and pitch format. The panodata in the [Equirectangular docs ](https://photo-sphere-viewer.js.org/guide/config.html#advanced-options) show poseHeading as a number from 0 to 360. Is there a way to get that value so I can store it and later set it?

**Comment by mistic100** [comment]
heading = yaw / Math.PI * 180

simple as that

**Reply by mistic100** [reply]
Do not mix up poseHeading which is inherent to the panorama file, and yaw which is the camera position (it starts at `defaultYaw` and increase/decrease when the user moves).

**Reply by AndyBaird** [reply]
Thanks, that points me in a better direction. Can defaultYaw be specified per node using the virtualTour plugin?

**Reply by mistic100** [reply]
No, the plugin is meant to keep the orientation of the user when navigating between nodes.

If your panoramas files are not all oriented in the same direction you need to specify `sphereCorrection.pan`
